.travis.yml

```diff
@@ -116,13 +116,8 @@ install:
   - if [[ "`uname`" == "Linux" ]]; then travis_retry pip3 install --user gitpython; fi
 
   # Install latest Polly toolchains and scripts
-<<<<<<< HEAD
-  - wget --quiet https://github.com/ruslo/polly/archive/master.zip
-  - unzip -q master.zip
-=======
```

Comment:
I'm not sure why I get into this merge conflict trouble, but I think I fixed it on a correct way, or? Please verify. Thanks

Reply:
Your fix is correct. Just make sure to create your branch off of the latest pkg.template instead of master